assemblies/assembly-install-rhdh-ocp.adoc

```diff
@@ -5,19 +5,33 @@
 
 toc::[]
 
-You can install {product} on {ocp-short} by using one of the following methods:
+You can install {product} on {ocp-short} by using one of the following installers:
 
-* The Helm chart
 * The {product} Operator
+* The Helm chart
 
-// Helm chart method
-include::modules/installation/proc-install-rhdh-ocp-helm.adoc[leveloffset=+1]
-include::modules/installation/proc-installing-rhdh-on-openshift-with-helm-cli.adoc[leveloffset=+2]
-include::modules/getting-started/proc-add-custom-app-file-openshift-helm.adoc[leveloffset=+2]
-include::modules/installation/proc-install-rhdh-airgapped-environment-ocp-helm.adoc[leveloffset=+2]
+== Installing {product} on {ocp-short} by using the operator
+
+You can install {product} on {ocp-short} by using the operator with one of the following methods:
 
-// Operator method
-include::modules/installation/proc-install-rhdh-ocp-operator.adoc[leveloffset=+1]
+* The {ocp-short} console
+* The {ocp-short} CLI
+
+include::modules/admin/proc-install-operator.adoc[leveloffset=+2]
```

Comment:
Another reason we were going to leave this procedure in the Admin guide and link to it from the Prerequisite sections of all Operator-related procedures is because its not specific to any one platform and it is a little weird to see content about K8s under an OCP-specific chapter. 

One way to resolve this is by making it a higher `leveloffset` so that it comes BEFORE any platform-specific Operator procedures rather than repeating it as part of each platform-specific Operator proc. 

So something like 

- Installing RHDH with the Operator
- - Installing the RHDH Operator
- - Installing RHDH on OCP with the Operator (include installing the Operator as a Prereq) 
- - Installing RHDH on EKS with the Operator (include installing the Operator as a Prereq) 
- - Installing RHDH on AKS with the Operator (include installing the Operator as a Prereq)

Reply:
Nevermind, scratch that - We want to organize these chapters by Platform, not by installer method. 

Leaving the _Installing the RHDH Operator_ section in the Admin guide and linking to it from the relevant Prereq sections in the installation guide might still be the way to go, at least for the first iteration.

Reply:
Indeed, the procedure is clearly with only OCP in mind! Removing from EKS and AKS assemblies, where the OpenShift console is not available.